pillowtop/listener.py

```diff
@@ -645,20 +662,6 @@ def change_transport(self, doc_dict):
         domain and case type
         """
         try:
-            if not self._type_exists(doc_dict):
```

Comment:
this is the main change - i think back in the day we used to have multiple mappings per pillow dependent on other properties of the document, but that has disappeard and this code was unnecessarily complicated